app/context.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This isn't strictly needed. Adding it to avoid coupling the `api4` package with `sqlstore`.